README.md

```diff
@@ -344,7 +344,14 @@ Finally, you're ready to run a computation using `mpirun`:
 ```bash
 mpirun -hostfile hostfile -n 3 ./main -m ./models/7B/ggml-model-q4_0.gguf -n 128
 ```
+### Vulkan Build
 
+**Vulkan** is a new library aimed at replacing **OpenGL**. Starting from version `b1996`, `llama.cpp` now supports **Vulkan** as a backend. It can be faster (almost 2x on NVIDIA GeForce RTX™ 3060 Laptop GPU) than **CUDA** on a consumer-grade NVIDIA graphics card.
```

Comment:
This is just not true. If the vulkan backend is faster than CUDA, you have probably a configuration issue.

Reply:
> Vulkan is a new library aimed at replacing OpenGL

OpenGL and OpenCL are considered deprecated on Mac. I couldn't find anything else related to this. Nothing I could find or see on the official sites or docs either. 

> It can be faster (almost 2x on NVIDIA GeForce RTX™ 3060 Laptop GPU) than CUDA

Why is a comparison being made here? Let people perform their own benchmarks and make comparisons on their own. There's not enough evidence to back this up and is new and experimental.